File: 1878-check-if-array-is-sorted-and-rotated/1878-check-if-array-is-sorted-and-rotated.cpp
 

class Solution {
public:
    bool check(vector<int>& nums) {
        int count = 0;
        for(int i=0;i<nums.size()-1;i++){
            if(nums[i]>nums[i+1]){
                count++;
            }
        }
        if(count>1){
            return false;
        }
        if(count>=1 && nums[0]<nums[nums.size()-1]){
            return false;
        }

        return true;
    }
};

File: 1878-check-if-array-is-sorted-and-rotated/1878-check-if-array-is-sorted-and-rotated.py
class Solution(object):
    def check(self, nums):
        """
        :type nums: List[int]
        :rtype: bool
        """
        n=len(nums)
        count=0
        for i in range(0,n):
            if (nums[i]>nums[(i+1)%n]):
                count=count+1

        return count<=1
        